bpf/bpf_lxc.c

```diff
@@ -1272,6 +1272,11 @@ static __always_inline int __tail_handle_ipv4(struct __ctx_buff *ctx,
 		return DROP_FRAG_NOSUPPORT;
 #endif
 
+#ifdef ENABLE_MAC_SPOOF_CHECK
```

Comment:
Use `DISABLE_SMAC_VERIFICATION` instead? to be consistent with `DISABLE_SIP_VERIFICATION`.

Reply:
As a general rule, using `ENABLE_FOO` is much easier to reason about from a developer perspective than `DISABLE_FOO`.